synapse/config/server.py

```diff
@@ -1104,11 +1121,22 @@ def generate_config_section(
           #  type: manhole
 
         # Connection settings for the manhole
+        #
         manhole_settings:
-        #  username: matrix
-        #  password: rabbithole
-        #  ssh_priv_key: CONFDIR/id_rsa
-        #  ssh_pub_key: CONFDIR/id_rsa.pub
+
+          # The username for the manhole. This defaults to matrix
+          #
+          # username: matrix
```

Comment:
per https://matrix-org.github.io/synapse/develop/code_style.html#configuration-file-format: no space before `username`, and `matrix` should be something *other than* the default.